filters/utilization_threshold.cpp

```diff
@@ -97,6 +97,7 @@ Try<Nothing> UtilizationThresholdFilter::consume(const ResourceUsage& product) {
     // Send only when node utilization is not too high.
     if ((totalCpuUsage / totalSlaveCpus.get()) < this->utilizationThreshold) {
       // Continue pipeline.
+      SERENITY_LOG(INFO) << "Producing.";
```

Comment:
We are filling the logs tremendously fast; can we do this in a debug level or at least include more context about the particular call site? ('Producing what?')